docs/extras/tactic_writing.md

```diff
@@ -434,15 +434,27 @@ This section is a direct compilation of messages from Mario on Zulip.
 * ``` ``(my pexpr) ``` constructs a pre-expression at parse time, resolving in the current (of the tactic) namespace
 * ```` ```(my pexpr) ```` constructs a pexpr, but defers resolution to run time (of the tactic), meaning that any references will be resolved in the namespace of the begin end block of the user, rather than the tactic itself
 * `%%`: This is called anti-quotation, and is supported in all the expr and pexpr quoting expressions `` `(expr) ``, ``` ``(pexpr) ```, ```` ```(pexpr) ````, as well as `` `[tacs] ``. Wherever an expression is expected inside one of these quoting constructs, you can use `%%e` instead, where `e` has type `expr` in the outer context of the tactic, and it will be spliced into the constructed expr/pexpr/etc. For example, if `a b : expr` then  `` `(%%a + %%b) `` is of type `expr`
+* ``%%`(n)``: To refer to local variables from the tactic definition inside a quotation, use ``%%`(n)``. As an example, we could write
+```
```

Comment:
Could you  indent this so that it belongs to the list?

Reply:
Is this the idiom we want to promote over `%%(reflect n)`? I believe they have the same behavior. The nested quotations look strange to my eye, and I think it obscures what's going on.

Reply:
Rob, I think @semorrison  wrote that right after discovering that it works. Please don't hesitate to write something more idiomatic